calendar-bundle/src/Resources/contao/models/CalendarEventsModel.php

```diff
@@ -230,21 +234,31 @@ public static function findPublishedByParentAndIdOrAlias($varId, $arrPids, array
 	/**
 	 * Find events of the current period by their parent ID
 	 *
-	 * @param integer $intPid     The calendar ID
-	 * @param integer $intStart   The start date as Unix timestamp
-	 * @param integer $intEnd     The end date as Unix timestamp
-	 * @param array   $arrOptions An optional options array
+	 * @param integer $intPid      The calendar ID
+	 * @param integer $intStart    The start date as Unix timestamp
+	 * @param integer $intEnd      The end date as Unix timestamp
+	 * @param array   $arrOptions  An optional options array
+	 * @param string  $blnFeatured If true, return only featured events, if false, return only unfeatured events
 	 *
 	 * @return Collection|CalendarEventsModel[]|CalendarEventsModel|null A collection of models or null if there are no events
 	 */
-	public static function findCurrentByPid($intPid, $intStart, $intEnd, array $arrOptions=array())
+	public static function findCurrentByPid($intPid, $intStart, $intEnd, array $arrOptions=array(), $blnFeatured = null)
```

Comment:
`$arrOptions` needs to remain the last argument, therefore I think we should add the featured option to `$arrOptions` instead. Similar to

https://github.com/contao/contao/blob/92ff7894b8bacc8b498b297fbedd7f5b7cb6a295/core-bundle/src/Resources/contao/models/PageModel.php#L716-L719

Reply:
Because BC?

Reply:
No, adding another optional argument would not break BC. But since we already have `$arrOptions`, we should use it. 😄

Reply:
alrighty then 😎